packages/connectivity/src/scp-cf/environment-accessor/service-credentials.ts

```diff
@@ -40,16 +40,9 @@ export function getServiceCredentials<
       typeof token === 'string' ? decodeJwt(token) : token
     );
 
-    if (credentials) {
-      return credentials;
-    }
-
-    logger.debug(
-      'Could not match binding service instance using JWT client id or audience.'
-    );
-  } else {
-    logger.debug(`No JWT given to select binding to service '${service}.`);
+    return credentials ?? undefined;
```

Comment:
Is this ever `null`? If so, why? If not, why do we need the `??` check? (Did I suggest this in the review? ;))

Reply:
Good catch, I assumed one of the 2 functions which can be called could return null, but it could be undefined at most, I guess I can remove it (no you didn't suggest it 😂)